src/utils/numbers/index.ts

```diff
@@ -29,4 +29,6 @@ const toPercent = (value: string | number): number => {
   return parseFloat(value) * 100;
 };
 
-export { toDecimal, toPercent };
+const convertZeroToOneIndex = (value: number): number => value + 1;
```

Comment:
`formatForDisplay`? and add a jsdoc comment t
```suggestion
/**
 * `formatForDisplay` - Formats a zero-indexed number for display in the UI.
 */
const formatForDisplay = (value: number): number => value + 1;
```